### PR TITLE
Remove Slashes from Datalog vars

### PR DESCRIPTION
**Pull request by dvargas92495**

### CODE REVIEW

File: src/utils/compileDatalog.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import type {
   DatalogClause,
 } from "roamjs-components/types/native";
 
-const toVar = (v = "undefined") => v.replace(/[\s"()[\]{}]/g, "");
+const toVar = (v = "undefined") => v.replace(/[\s"()[\]{}/\\]/g, "");
```
Comment (dvargas92495) [inline src/utils/compileDatalog.ts:7]: can you pull and verify it this works for those queries?

**Review by mdroidian — CHANGES_REQUESTED** [review]
`Unmatched delimiter ].`

**Comment by dvargas92495** [comment]
Can you paste in the query that's resulting in that error? I'm failing to repro

**Comment by mdroidian** [comment]
```
Find node Where
node is a Question
```

```
[:find
  (pull ?node [:block/string :node/title :block/uid])
  (pull ?node [:block/uid])
:where
  [(re-pattern "^\\[\\[QUE\\]\\] - (.*?)$") ?/^\\QUE\\-.*?$/-regex]
  [?node :node/title ?node-Title]
  [(re-find ?/^\\QUE\\-.*?$/-regex ?node-Title)]
]
```

**Comment by mdroidian** [comment]
Sorry, posted wrong query.  Edited previous comment.

**Comment by dvargas92495** [comment]
The problem seems to be in the string, not the getVar function. Namely:

`"^\\[\\[QUE\\]\\] - (.*?)$"`

should be

`"^\\\\[\\\\[QUE\\\\]\\\\] - (.*?)$"`

**Comment by dvargas92495** [comment]
Is it not hitting this replace? https://github.com/dvargas92495/roamjs-query-builder/blob/SLASHES/src/utils/conditionToDatalog.ts#L86

**Comment by mdroidian** [comment]
Sorry, I can't seem to test it right now.  I'm having some weird issue.

Whenever I load query builder extension via `Developer Extensions` I'm was getting multiple of the following error:
`ERROR in load-extension: invalid ext-id:  null`

I loaded up a new graph and get this error:
`app-graph.js:68 Uncaught (in promise) Error: Can not register `navigator` for `core` since `navigator` already exists in the prototype and can not be overridden`

And Chrome doesn't register query builder in sources
image

I've tried your branch, my fork, and an old commit on my fork. 🤷‍♂️

To top it off, our power it out, so my laptop is about to die. 😣

**Comment by mdroidian** [comment]
running `npx roamjs-scripts dev --depot` instead of `npm start` seemed to do it

**Comment by mdroidian** [comment]
> Is it not hitting this replace? https://github.com/dvargas92495/roamjs-query-builder/blob/SLASHES/src/utils/conditionToDatalog.ts#L86

I believe it is hitting it
image

**Comment by dvargas92495** [comment]
Ok I was able to reproduce both issues:
- delimiter was solved by adding the `^` to the exclusion regex
- build issue solved by bumping up roamjs-scripts to latest

**Review by mdroidian — APPROVED** [review]

**Pull request merged**